### DISCUSSION TITLE
Allow user to track mood (and other categorical variables)

### DISCUSSION THREAD

**Discussion by NicoCaro** [iSoron/uhabits#409] [Feature Requests]
I'd like to have a *mood tracker* special habit. I think it's already possible to create an habit to represent one's mood but this representation fails to capture "mood oscillations" (because of the score algorithm for habits). So this mood tracker should only plot the current and past *mood values* without any transformation. I've seen this in apps like *remente* and I find it very useful.

**Comment by iSoron** [comment]
I agree that this would be a nice feature to have. The current plan is to finish the work on numerical habits (issue #42). After that feature is working well, we can start working on variations such as the one you propose (e.g. How was your day / How is your mood? Excellent, good, acceptable, poor, terrible?).

**Reply by auipga** [reply]
#42 is completed (and nice). Can we reactivate this idea?

Similar #979
